### ISSUE TITLE
Create dataset loader for SciQ

### ISSUE THREAD

**Issue by jason-fries** [bigscience-workshop/biomedical#251]
## Adding a Dataset
- **Name:** SciQ
- **Description:** *None provided*
- **Task:** QA
- **Paper:** https://aclanthology.org/W17-4413/
- **Data:** https://allenai.org/data/sciq
- **License:** ?

**Comment by j-chim** [comment]
#self-assign

**Comment by jason-fries** [comment]
Hi @j-chim  can you let us know if you are still working on this so we can update our project board? Please just notify us the status **by Friday April 8**.  You can response to this comment or ping us on Slack or Discord. 

No worries if you are not finished but still intend to work on this!

**Comment by j-chim** [comment]
Yup, thanks for checking in! doing this over at #341. Just waiting for the multiple choice QA schema to be finalized before I push an updated version.
